MATLAB/dynamics_update.m

```diff
@@ -82,16 +82,21 @@
         quat_body_eci=y(7:10);
         quat_eci_body= utl_quat_conj(quat_body_eci);
         quat_body_ecef= utl_quat_cross_mult(quat_body_eci,quat_eci_ecef);
-        pos_eci=y(1:3);
+        pos_eci=y(1:3); 
+        vel_eci=y(4:6);
         pos_ecef=utl_rotateframe(quat_ecef_eci,pos_eci);
+        vel_ecef = utl_rotateframe(quat_ecef_eci,vel_eci);
```

Comment:
Unsure exactly how we decided on doing the velocity conversion, but we may not be accounting for the velocity component due to ECEF's rotation in ECI (something like `cross(w, r)`). If we don't want to account for this term we should call `vel_ecef` something different.

Reply:
You can get earth's rotation rate from `const.earth_rate_ecef` Then add something like 
```suggestion
        vel_ecef = utl_rotateframe(quat_ecef_eci,vel_eci);
        vel_ecef= vel_ecef - cross(const.earth_rate_ecef,pos_ecef);
```